### PR TITLE
Added release notes for 1.9.2

### PR DESCRIPTION
**Pull request by nacx**
Added release notes for 1.9.2.

I've pushed the branch to the ASF repos, so all committers can directly modify / fix / add the missing bits to them.

### CODE REVIEW

**Comment by jclouds-commentator** [comment]
Go to http://5a95012866f326a39535-1f1c5fe3b2b50be8f5ad3ddf2e64fae5.r45.cf5.rackcdn.com/ to review your changes.

**Comment by jclouds-commentator** [comment]
Go to http://51044e6c6c7e61ef60d2-e9f1c0666d7a3603e317e772889711b1.r97.cf5.rackcdn.com/ to review your changes.

**Comment by andrewgaul** [comment]
No blobstore notes for this release.

**Comment by andreaturli** [comment]
+1 thanks Ignasi

File: releasenotes/index.md [hunk 1]
```diff
@@ -6,6 +6,7 @@ permalink: /releasenotes/
 
 The Release Notes and Javadocs for every major release of Apache jclouds are listed chronologically below.
 
+* [1.9.2](/releasenotes/1.9.2) | [Javadoc](/reference/javadoc/1.9.x/)
```
Comment (demobox) [inline releasenotes/index.md]: Have we published the 1.9.2 Javadoc yet? It looks like we might have even forgotten to publish 1.9.1 ;-)

http://jclouds.apache.org/reference/javadoc/1.9.x/

File: releasenotes/1.9.2.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+permalink: /releasenotes/1.9.2/
+---
+
+1. [Introduction](#intro)
+1. [Credits](#credits)
+1. [Test Results](#test)
+
+## <a id="intro"></a>Introduction
+
+You can read the official announcement at [(TODO) Apache jclouds 1.9.2 released](http://markmail.org/message/7axjkmb7kypgvtot). To get started with jclouds, please see the [jclouds installation guide](/start/install/).
```
Comment (demobox) [inline releasenotes/1.9.2.md]: Still a TODO here, I guess?

Comment (nacx) [inline releasenotes/1.9.2.md]: Yes, I was waiting for the mirrors to send the annoincement. I'll update the link in a while.

File: releasenotes/index.md (hunk 1 above)
Comment (nacx) [inline releasenotes/index.md]: Looking at other releases, it looks like we don't publish javadocs for every minor?
http://jclouds.apache.org/reference/javadoc/

Should we deploy all missing versions?

Comment (demobox) [inline releasenotes/index.md]: > Should we deploy all missing versions?

Good question. I think we have a couple of options here:

* Point the A.B.x Javadoc link to A.B.0, which seems to be the case with 1.9.x now. But then we should probably change the link from `A.B.x` to `A.B.0`?
* Update the A.B.x Javadoc to point to the latest release, i.e. overwrite the 1.9.0 Javadoc with 1.9.2, and keep the `1.9.x` link. This seems most consistent with the name of the link
* Publish Javadoc for all the versions, and then change the links accordingly

@nacx What do you think..?

Comment (nacx) [inline releasenotes/index.md]: Ideally option 3, but I recall some discussions about the amount of space required to host that, so given that most javadoc will not change in minor versions (at least it shouldn't) I'd say we should go for option 2.

**Comment by jclouds-commentator** [comment]
Go to http://c69ee843fc5c3c74c59a-f878829ce17ae283dad2b9ab632cb576.r34.cf5.rackcdn.com/ to review your changes.

**Comment by nacx** [comment]
Squashed and pushed to master as [56b1ebaf](http://git-wip-us.apache.org/repos/asf/jclouds-site/commit/56b1ebaf).

File: releasenotes/index.md (hunk 1 above)
Comment (demobox) [inline releasenotes/index.md]: > javadoc will not change in minor versions (at least it shouldn't) I'd say we should go for option 2

+1 - sounds good to me!

**Comment by nacx** [comment]
Site and javadocs (option 2) published too. Thanks!

**Pull request closed without merging**